core/src/main/java/bisq/core/provider/mempool/TxValidator.java

```diff
@@ -355,17 +431,69 @@ private Coin getTakerFeeRateBtc(long blockHeight) {
         return daoStateService.getParamValueAsCoin(Param.DEFAULT_TAKER_FEE_BTC, (int) blockHeight);
     }
 
+    private Optional<Boolean> maybeTestFeeFromFilter(Coin tradeAmount,
```

Comment:
Maybe call it `maybeCheckAgainstFeeFromFilter`? `maybeTestFeeFromFilter` sounds a little bit weird to me (I'm no native speaker)

Reply:
Done